homeassistant/components/iotawatt/sensor.py

```diff
@@ -89,7 +90,7 @@ class IotaWattSensorEntityDescription(SensorEntityDescription):
     ),
     "VAR": IotaWattSensorEntityDescription(
         key="VAR",
-        native_unit_of_measurement=VOLT_AMPERE_REACTIVE,
+        native_unit_of_measurement=UnitOfReactivePower.VOLT_AMPERE_REACTIVE,
```

Comment:
Is this a breaking change for `iotawatt`?
new is `var` (lower case) vs old `VAR` (upper case)

Maybe it should be added to "corresponding units" (unless that is already case insensitive)?

Reply:
I think we should revert this change for the iotawatt integration. If we want to change this it should go in a separate PR.